src/hal/src/lib.rs

```diff
@@ -242,21 +242,28 @@ bitflags! {
         const UNSIZED_DESCRIPTOR_ARRAY = 0x0800_0000_0000_0000;
 
         /// Support triangle fan primitive topology.
-        const TRIANGLE_FAN = 0x0001 << 64;
+        const TRIANGLE_FAN = 0x0000_0000_0000_0001 << 64;
         /// Support separate stencil reference values for front and back sides.
-        const SEPARATE_STENCIL_REF_VALUES = 0x0002 << 64;
+        const SEPARATE_STENCIL_REF_VALUES = 0x0000_0000_0000_0002 << 64;
         /// Support manually specified vertex attribute rates (divisors).
-        const INSTANCE_RATE = 0x0004 << 64;
+        const INSTANCE_RATE = 0x0000_0000_0000_0004 << 64;
         /// Support non-zero mipmap bias on samplers.
-        const SAMPLER_MIP_LOD_BIAS = 0x0008 << 64;
+        const SAMPLER_MIP_LOD_BIAS = 0x0000_0000_0000_0008 << 64;
 
         /// Make the NDC coordinate system pointing Y up, to match D3D and Metal.
-        const NDC_Y_UP = 0x0001 << 80;
+        const NDC_Y_UP = 0x0000_0000_0001_0000 << 64;
 
         /// Supports task shader stage.
-        const TASK_SHADER = 0x0001 << 96;
+        const TASK_SHADER = 0x0000_0001_0000_0000 << 64;
         /// Supports mesh shader stage.
-        const MESH_SHADER = 0x0002 << 96;
+        const MESH_SHADER = 0x0000_0002_0000_0000 << 64;
+
+        /// Supports multiview
+        const MULTIVIEW = 0x0001_0000_0000_0000 << 64;
```

Comment:
this should be `0x0000_0004_0000_0000` and so on. No need to leave such a big gap with the mesh shader